rmf_fleet_adapter/launch/fleet_adapter.launch.xml

```diff
@@ -27,7 +27,7 @@
   <arg name="perform_deliveries" default="false" description="Whether this fleet adapter can perform deliveries"/>
   <arg name="perform_cleaning" default="false" description="Whether this fleet adapter can perform cleaning"/>
   <arg name="finishing_request_charge_battery" default="false" description="Whether robots in this fleet should perform a ChargeBattery task once all queued tasks are completed"/>
-  <arg name="finishing_request_return_charger" default="false" description="Whether robots in this fleet should return to their chargers once all queued tasks are completed"/>
+  <arg name="finishing_request_park_robot" default="false" description="Whether robots in this fleet should return to their parking spots once all queued tasks are completed"/>
```

Comment:
Sorry for just thinking of this now, but I wonder if we should refactor these finishing request parameters to just one:

```
<arg name="finishing_request" default="nothing" description="What should happen when the robot is finished with its tasks? Options: nothing (default), park, charge"/>
```

As it is now, it's kind of confusing what might happen if `finishing_request_charge_battery` and `finishing_request_park_robot` are both `true`.

Reply:
If a user passes an unrecognized string, we can just print out a warning and default to `nothing` behavior.

Reply:
Fair point. But how would users know what the valid strings are for different factories? It seems that the mapping will be an implementation detail.

Reply:
Right, that's why I recommend putting all the supported strings into the parameter description (see the `Options: ...` at the end of the code snippet that I recommended).

Off the top of my head, I don't know how visible that parameter description will be to end-users. But we could also print out the list of supported parameters in the warning that we issue when we receive an invalid string.

Reply:
Roger. I've made the requested change!